approaching_emergency_vehicle_plugin/test/test_approaching_erv.cpp

```diff
@@ -938,6 +938,78 @@ namespace approaching_emergency_vehicle_plugin{
     }
 
+    TEST(Testapproaching_emergency_vehicle_plugin, testApproachingErvStatusMessage){
+
+        // Create, configure, and activate worker_node (ApproachingEmergencyVehiclePlugin)
+        rclcpp::NodeOptions options;
+        auto worker_node = std::make_shared<approaching_emergency_vehicle_plugin::ApproachingEmergencyVehiclePlugin>(options);
+
+        worker_node->configure(); //Call configure state transition
+        worker_node->activate();  //Call activate state transition to get not read for runtime
+
+        // Generate status message after initial node activation (no ERV is being tracked) and verify the output
+        carma_msgs::msg::UIInstructions status_msg = worker_node->generateApproachingErvStatusMessage();
+        ASSERT_EQ(status_msg.type, carma_msgs::msg::UIInstructions::INFO);
+        ASSERT_EQ(status_msg.msg.substr(0,22), "HAS_APPROACHING_ERV:0,");
+
+        // Generate status message when an ERV is being tracked in state MOVING_OVER_FOR_APPROACHING_ERV (left lane change) and verify the output
+        worker_node->has_tracked_erv_ = true;
+        worker_node->tracked_erv_.seconds_until_passing = 11.56342;
+        worker_node->transition_table_.event(ApproachingEmergencyVehicleEvent::APPROACHING_ERV_IN_PATH);
+        worker_node->has_planned_upcoming_lc_ = true;
+        worker_node->upcoming_lc_params_.is_right_lane_change = false;
+
+        status_msg = worker_node->generateApproachingErvStatusMessage();
+        ASSERT_EQ(status_msg.type, carma_msgs::msg::UIInstructions::INFO);
+        ASSERT_EQ(status_msg.msg, "HAS_APPROACHING_ERV:1,TIME_UNTIL_PASSING:11.5634,EGO_VEHICLE_ACTION:Changing lanes to the left.");
+
+        // Generate status message when an ERV is being tracked in state MOVING_OVER_FOR_APPROACHING_ERV (right lane change) and verify the output
+        worker_node->has_planned_upcoming_lc_ = true;
+        worker_node->upcoming_lc_params_.is_right_lane_change = true;
+
+        status_msg = worker_node->generateApproachingErvStatusMessage();
+        ASSERT_EQ(status_msg.type, carma_msgs::msg::UIInstructions::INFO);
+        ASSERT_EQ(status_msg.msg, "HAS_APPROACHING_ERV:1,TIME_UNTIL_PASSING:11.5634,EGO_VEHICLE_ACTION:Changing lanes to the right.");
+
+        // Generate status message when an ERV is being tracked in state WAITING_FOR_APPROACHING_ERV and verify the output
+        worker_node->has_tracked_erv_ = true;
+        worker_node->tracked_erv_.seconds_until_passing = 11.563429203;
+        worker_node->transition_table_.event(ApproachingEmergencyVehicleEvent::ERV_ABOUT_TO_PASS_IN_PATH);
+
+        status_msg = worker_node->generateApproachingErvStatusMessage();
+        ASSERT_EQ(status_msg.type, carma_msgs::msg::UIInstructions::INFO);
+        ASSERT_EQ(status_msg.msg, "HAS_APPROACHING_ERV:1,TIME_UNTIL_PASSING:11.5634,EGO_VEHICLE_ACTION:Remaining in the current lane at the speed limit.");
```

Comment:
What is the speed limit value? Maybe display that on the UI as well (converted to MPH)?